kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/dao/EmployeeDao.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.hibernate.manytomany.dao;
+
+import com.kodilla.hibernate.manytomany.Company;
+import com.kodilla.hibernate.manytomany.Employee;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+
+@Transactional
+@Repository
+public interface EmployeeDao extends CrudRepository<Company, Integer> {
+
+    @Query(nativeQuery = true)
+    List<Employee> retrieveEmployeeWithThreeChars(@Param("LASTNAME") String lastname);
```

Comment:
Czemu akurat `ThreeChars`? Nigdzie w kodzie nie jest to wymuszone, że muszą być akurat 3.